src/Microsoft.AspNet.Mvc.Core/ControllerActionExecutor.cs

```diff
@@ -125,9 +126,19 @@ private static Task<object> Convert<T>(object taskAsObject)
                     }
                     else
                     {
-                        value = parameterInfo.ParameterType.GetTypeInfo().IsValueType
-                            ? Activator.CreateInstance(parameterInfo.ParameterType)
-                            : null;
+                        var defaultValueAttribute = 
+                            parameterInfo.GetCustomAttribute<DefaultValueAttribute>(inherit: false);
```

Comment:
mm.. do we want to not see the inherited attributes? Is it because of compat or are we making a call here?